tools/skilldbconverter.php

```diff
@@ -654,7 +654,7 @@ function getnk($nk)
 {
 	$bitmask = array(
 		"NoDamage"         =>   intval(0x01), //- No damage skill
-		"SplashArea"       =>   intval(0x02)+intval(0x04), //- Has splash area (requires source modification)
```

Comment:
1- it's not pointless, but yes it can be prettier (which will happen later when this fix finished)
2- this check you made caused skills to have been defined as splasharea by wrong, many skills is bugged.